docs/native-modules.md

```diff
@@ -48,6 +48,8 @@ Open the Visual Studio solution in the `windows` folder and add the new files di
 | `ReactConstant`         | Specifies a field or property that represents a constant. |
 | `ReactConstantProvider` | Specifies a method that provides a set of constants.      |
 | `ReactEvent`            | Specifies a field or property that represents an event.   |
+| `ReactInit`             | Specifies a class initialization method.                  |
+| `ReactFunction`         | Specifies a field that helps calling a function.          |

```

Comment:
what does "a field that helps calling a function" mean? when would it be used? how is react function different from react method?

Reply:
From the [header](https://github.com/microsoft/react-native-windows/blob/0dfe9decb4390c4b77e0837e69c4a25878034960/vnext/Microsoft.ReactNative.Cxx/NativeModules.h) it looks like a std::function. Or in C# I assume it's a Func/Action/whatever? Anyway, I wasn't sure how much detail to go into. Really these should all link to a page that has the full detail that's in that header.

Reply:
The name is a bit confusing. Many engineers are going to use the terms method and function interchangeably. Maybe this should have been `ReactDelegate`?